prplmesh-be: Implement prplmesh_add/del.lua

VladyslavTupikin opened the pull request:
Need to add new backend scripts for creating and removing objects from the UBus.

Closes #3

MuteSpirit commented on src/prplmesh_add.lua:
```diff
@@ -0,0 +1,90 @@
+################################################################################
+--]]
+
+require("prplmesh_get")
```
Put `call_ubus` into separate module then `prplmesh_get`

VladyslavTupikin replied:
Done

MuteSpirit commented on src/prplmesh_del.lua:
```diff
@@ -0,0 +1,90 @@
+        ing.utils.exit(ing.ResCode.FAIL, {})
+    end
+
+    ing.utils.exit(ing.ResCode.SUCCESS, {})
```
I think this call is not enough to return data, needed for MMX
that should be almost
```
ing.utils.exit(ing.ResCode.SUCCESS, ing.StatCode.OK)
```

VladyslavTupikin replied:
fixed

MuteSpirit commented on src/prplmesh_add.lua:
```diff
@@ -0,0 +1,90 @@
+        ing.utils.exit(ing.ResCode.FAIL, {})
+    end
+
+    ing.utils.exit(ing.ResCode.SUCCESS, {})
```
According to examples from `user-be` and `host-be` script should print something like
```
0; 0; 1;
```
and that may be done by call
```
ing.utils.exit(ing.ResCode.SUCCESS, ing.StatCode.OK, INDEX-OF-JUST-ADDED-OBJECT)
```

VladyslavTupikin replied:
fixed

Commit:
prplmesh-be: Add prplmesh lua library

Created new script prplmesh-be-utils.lua
which contains 2 functions for log errors
into stderr and function for call ubus methods.

Also modified Makefile for installing library
script to the /usr/lub/lua folder.

Signed-off-by: Vladyslav Tupikin <v.tupikin@inango-systems.com>

Commit:
prplmesh-be: Add frames for add/del scripts

Added frames for add/del scripts.

Modified Makefile for installing new
scripts to the /usr/bin/mmx_be folder.

Signed-off-by: Vladyslav Tupikin <v.tupikin@inango-systems.com>

Commit:
prplmesh-be: Implement prplmesh_add.lua

Add implementation for add_object()
function.

Signed-off-by: Vladyslav Tupikin <v.tupikin@inango-systems.com>

Commit:
prplmesh-be: Implement prplmesh_del.lua

Add implementation for del_object()
function.

Signed-off-by: Vladyslav Tupikin <v.tupikin@inango-systems.com>

MuteSpirit commented on src/prplmesh_del.lua:
```diff
@@ -0,0 +1,118 @@
+    local idx = string.match(string.sub(path, -1),"[1-9]")
+
+    return tostring(ret) .. ";" .. tostring(ing.StatCode.OK) .. ";" .. tostring(idx) ..";"
+--add_object()
```
```Lua
--del_object
```

MuteSpirit commented on src/prplmesh_add.lua:
```diff
@@ -0,0 +1,124 @@
+        return ret
+    end
+
+    local mmx = add_object(args[1])
```
Add task to finish `add` script because this script must also accept and handle options line `-pname Parameter1Name -pvalue Parameter1Value -pname Parameter2Name -pvalue Parameter2Value ...`
